cmd/xdpcap/main.go

```diff
@@ -166,6 +185,10 @@ func capture(mapPath string, pcapFile *os.File, quiet bool, filterExpr string, o
 					fmt.Fprintln(os.Stderr, "Error writing packet:", err)
 				}
 
+				if opts.flush {
+					pcapWriter.Flush()
```

Comment:
Abort on error?

Reply:
Good catch. I added a log message instead of aborting as that's what we do for `pcapWriter.WritePacket` already.